### PR TITLE
Add room finder to more view

### LINKED ISSUE #34: Add RoomFinder to "More" View
Currently the only way to search for rooms is by using the search. A new user however doesn't know this feature exists.

There should be a reference in the more-view. I guess it might just open the search bar?

### PR DESCRIPTION
**Pull request by mammuth**
Before there was no hint to the user, that the room finder is implemented. Now it's referenced in the more-view.
SearchViewController is restricted to the RoomSearch when opened via the more view.

@mathiasquintero where are the current icons from? We probably should add a dedicated icon for the roomfinder?

Closes #34 

bildschirmfoto 2016-12-17 um 14 33 13

### CODE REVIEW

File: TUM Campus App/MoreTableViewController.swift [hunk 1]
```diff
@@ -76,7 +81,7 @@ extension MoreTableViewController {
             }
         case 5:
             UserDefaults.standard.removeObject(forKey: LoginDefaultsKeys.Token.rawValue)
-            let storyboard = UIStoryboard(name: "Main", bundle: nil)
+            let storyboard = UIStoryboard(name: "Setup", bundle: nil)
```
Comment (mathiasquintero) [inline TUM Campus App/MoreTableViewController.swift]: Wasn't this done before?

File: TUM Campus App/TumDataManager.swift [hunk 2]
```diff
@@ -77,9 +77,10 @@ class TumDataManager {
         }
     }
     
-    func search(_ receiver: TumDataReceiver, query: String) {
+    func search(_ receiver: TumDataReceiver, query: String, searchIn managersToSearchIn:[TumDataItems]? = nil) {
```
Comment (mathiasquintero) [inline TUM Campus App/TumDataManager.swift]: Why not = searchManagers? That way you don't have to check for an optional

File: TUM Campus App/MoreTableViewController.swift (hunk 1 above)
Comment (mammuth) [inline TUM Campus App/MoreTableViewController.swift]: Not in the MoreTableViewController. But somewhere else, yes.

File: TUM Campus App/TumDataManager.swift (hunk 2 above)
Comment (mammuth) [inline TUM Campus App/TumDataManager.swift]: Wanted to do so, but Xcode gave an error that it's not possible to use instance variables as default parameters.

Comment (mathiasquintero) [inline TUM Campus App/TumDataManager.swift]: And if we make it static? It makes sense for it to be static anyway

Comment (mathiasquintero) [inline TUM Campus App/TumDataManager.swift]: Let's leave it be for now

**Pull request merged**